src/components/Calendar.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div id="calendarFull">
+    <button @click="refreshEvents">Refresh</button>
+    <button v-if="selected._id" @click="removeEvent">Remove</button>
+    <pre v-if="selected._id">Title: {{selected.title}}
+    Start: {{selected.start}}
+    End: {{selected.end}}
+    Allday: {{selected.allDay}}
+    ID: {{selected.id}}</pre>
+    <full-calendar ref="calendar" :events="events" @event-drop="eventDrop" @event-resize="eventResize" @event-selected="eventSelected" @event-created="eventCreated" :config="config"></full-calendar>
+  </div>
+</template>
+
+<script>
+import frappe from "frappejs";
+import Vue from 'vue'
+import FullCalendar from 'vue-full-calendar';
+const { DateTime } = require('luxon');
+Vue.use(FullCalendar);
+
+const getCircularReplacer = () => {
+  const seen = new WeakSet;
+  return (key, value) => {
+    if (typeof value === "object" && value !== null) {
+      if (seen.has(value)) {
+        return;
+      }
+      seen.add(value);
+    }
+    return value;
+  };
+};
+
+export default {
+  name: 'calendarFull',
+  data() {
+    return {
+      events: [],
+
+      config: {
+        eventClick: (event) => {
+          this.selected = JSON.parse(JSON.stringify(event, getCircularReplacer()));
+        },
+      },
+
+      selected: {},
+    };
+  },
+
+  async created(){ 
```

Comment:
Please try to format your code using standard JS style

Reply:
Ok' i'll look into it, and revert asap.